src/dotnet/APIView/APIViewWeb/Languages/PythonLanguageService.cs

```diff
@@ -3,18 +3,28 @@
 
 using System;
 using System.IO;
+using Microsoft.Extensions.Configuration;
 
 namespace APIViewWeb
 {
     public class PythonLanguageService : LanguageProcessor
     {
-        private static readonly string PythonHome = Environment.GetEnvironmentVariable("APIVIEW_PYTHONHOME") ?? string.Empty;
-
+        private readonly string ApiViewPythonProcessor;
         public override string Name { get; } = "Python";
         public override string Extension { get; } = ".whl";
-        public override string ProcessName { get; } = Path.Combine(PythonHome, "Scripts", "apistubgen");
+        public override string ProcessName
+        {
+            get{ return ApiViewPythonProcessor; }
+        }
         public override string VersionString { get; } = "0.1.1";
 
+        public PythonLanguageService(IConfiguration configuration)
+        {
+            // apistubgen is located in python's scripts path e.g. <Pythonhome>/Scripts
+            var PythonHome = configuration["APIVIEW_PYTHONHOME"] ?? string.Empty;
```

Comment:
You don't need a `APIVIEW_` prefix, it gets trimmed from environment variables automatically.